ui/components/core/src/components/Histogram.svelte

```diff
@@ -15,105 +15,81 @@
    */
 
   // Imports
-  import { VegaLite, type VisualizationSpec } from "svelte-vega";
+  import { Bar } from "svelte-chartjs";
+  import {
+    Chart,
+    Title,
+    Tooltip,
+    Legend,
+    BarElement,
+    CategoryScale,
+    LinearScale,
+    type ChartDataset,
+  } from "chart.js";
+  import zoomPlugin from "chartjs-plugin-zoom";
+  import type { DatasetStat } from "..";
 
-  import type { DatasetStat } from "../lib/types/datasetTypes";
-
-  // Exports
+  // Define props
   export let hist: DatasetStat;
-  export let maxHeight = 48;
-  export let hideTitle = false;
 
-  // Calculate histogram height
-  let h = 7.5 * hist.histogram.length;
-  h = h < 125 ? 125 : h;
+  // Color palette for stacked charts
+  const colors = ["#771E5F", "#1E7736", "#1E5F77", "#77361E"];
```

Comment:
suggestion: it could be useful to keep those values in a setting file to re-use them later